packages/broadway.rb

```diff
@@ -16,58 +16,67 @@ class Broadway < Package
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> initbroadway"
-    system "echo 'if [ -z \"\${BROADWAYD}\" ]; then' >> initbroadway"
-    system "echo '  [ -f #{CREW_PREFIX}/bin/stopsommelier ] && stopsommelier' >> initbroadway"
-    system "echo '  broadwayd \${BROADWAY_DISPLAY} &' >> initbroadway"
-    system "echo '  sleep 3' >> initbroadway"
-    system "echo 'fi' >> initbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> initbroadway"
-    system "echo 'if [ ! -z \"\${BROADWAYD}\" ]; then' >> initbroadway"
-    system "echo '  echo \"broadwayd process \${BROADWAYD} is running\"' >> initbroadway"
-    system "echo 'else' >> initbroadway"
-    system "echo '  echo \"broadwayd failed to start\"' >> initbroadway"
-    system "echo '  exit 1' >> initbroadway"
-    system "echo 'fi' >> initbroadway"
-    system "echo '#!/bin/bash' > stopbroadway"
-    system 'echo >> stopbroadway'
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2>/dev/null)' >> stopbroadway"
-    system "echo 'if [ ! -z \"\${BROADWAYD}\" ]; then' >> stopbroadway"
-    system "echo '  pkill broadwayd' >> stopbroadway"
-    system "echo '  sleep 3' >> stopbroadway"
-    system "echo 'fi' >> stopbroadway"
-    system "echo 'BROADWAYD=\$(pidof broadwayd 2> /dev/null)' >> stopbroadway"
-    system "echo 'if [ -z \"\${BROADWAYD}\" ]; then' >> stopbroadway"
-    system "echo '  echo \"broadwayd process stopped\"' >> stopbroadway"
-    system "echo 'else' >> stopbroadway"
-    system "echo '  echo \"broadwayd process \${BROADWAYD} is running\"' >> stopbroadway"
-    system "echo '  exit 1' >> stopbroadway"
-    system "echo 'fi' >> stopbroadway"
+    @broadwayenv = <<~BROADWAYENVEOF
+      GDK_BACKEND=broadway
+      XDG_RUNTIME_DIR=/var/run/chrome
+      BROADWAY_DISPLAY=:5
+    BROADWAYENVEOF
+    File.write("#{CREW_DEST_HOME}/.broadway.env", @broadwayenv)
+    @initbroadway = <<~INITBROADWAYEOF
+      #!/bin/bash
+      BROADWAYD=$(pidof broadwayd 2>/dev/null)
+      if [ -z "${BROADWAYD}" ]; then
+        [ -f #{CREW_PREFIX}/bin/stopsommelier ] && stopsommelier
+        broadwayd ${BROADWAY_DISPLAY} &
+        sleep 3' >> initbroadway
+      fi
+      BROADWAYD=$(pidof broadwayd 2>/dev/null)
+      if [ ! -z "${BROADWAYD}" ]; then
+        echo "broadwayd process ${BROADWAYD} is running"
+      else
+        echo "broadwayd failed to start"
+        exit 1
+      fi
+    INITBROADWAYEOF
+    File.write('initbroadway', @initbroadway)
+    @stopbroadway = <<~STOPBROADWAYEOF
+      #!/bin/bash
+      BROADWAYD=$(pidof broadwayd 2>/dev/null)
+      if [ ! -z "${BROADWAYD}" ]; then
+        pkill broadwayd
+        sleep 3
+      fi
+      BROADWAYD=$(pidof broadwayd 2> /dev/null)
+      if [ -z "${BROADWAYD}" ]; then
+        echo "broadwayd process stopped"
+      else
+        echo "broadwayd process ${BROADWAYD} is running"
+        exit 1
+      fi
+    STOPBROADWAYEOF
+    File.write('stopbroadway', @stopbroadway)
```

Comment:
```
File.write("#{CREW_DEST_PREFIX}/bin/stopbroadway", @initbroadway)
```

Reply:
See the install section. (This applies to `cronie.rb` as well.)